.github/workflows/smoke-test.yaml

```
@@ -17,13 +28,27 @@ jobs:
         platform: [ubuntu-latest]
     runs-on: ${{ matrix.platform }}
-    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    if: >-
+      github.event_name == 'workflow_call' ||
+      github.event.workflow_run.conclusion == 'success'
     defaults:
       run:
         shell: bash
     steps:
+      - name: Check out code (workflow_call ref)
+        if: github.event.name == "workflow_call"
```

actionlint (1.7.12)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)